src/Entity/Timeline/Measure.php

```diff
@@ -312,22 +332,17 @@ public function profileIds(): array
     /**
      * @Algolia\Attribute
      */
-    public function titles(): array
+    public function manifestoId(): ?int
```

Comment:
maybe `getManifestoId()` with `@Algolia\Attribute(algoliaName="...")` if you want change the attribute name?